src/fragments/lib/graphqlapi/js/subscribe-data.mdx

```diff
@@ -40,6 +40,30 @@ Amplify.configure({
 });
 ```
 
+Subscriptions take an optional `filter` argument which provides an option to define and enable business logic for data filtering on the backend directly.
+
+```graphql
+// Subscribe to creation of Todo
+const subscription = API.graphql(
+    graphqlOperation(subscriptions.onCreateTodo, {
+      filter: {
+        type: { eq: "Personal" }
+      }
+    })
+).subscribe({
+    next: ({ provider, value }) => console.log({ provider, value }),
+    error: error => console.warn(error)
+});
+```
+
+Generally, don’t pass the `filter` parameter if you want to get all subscription events.
+
+<Callout>
+
+**Important**: Passing an empty object `{}` is NOT recommended because it means “no filter” only when multiple owner authorization is used.
```

Comment:
```suggestion
**Important**: Passing an empty object `{}` as a filter is NOT recommended because it means “no filter” only when multiple owner authorization is used.
```